docs/geedocs/5.3.5/sources/answer/getStarted/upgrade.rst.txt

```diff
@@ -71,6 +71,20 @@ Optionally enable cutter:  /opt/google/bin/gecutter enable
 
 Start gefusion and geserver.
 
+-[ Issues on upgrading ]-
+
+If you have previously published databases that use ExampleSearch and
+are upgrading to 5.3.5, you must install opengee-extra. Otherwise,
+ExampleSearch will stop working.
+
+If you are upgrading from 5.3.4.1 or before to 5.3.5 and do not
+install opengee-extra, ExampleSearch will still show up in the search
+tabs page of the admin console but it will not work (due to a bug).
+This is fixed in 5.3.6 and beyond.
```

Comment:
Should this be changed to "5.3.5.1 and beyond", or do we keep it to major versions?

Reply:
I don't know - @tst-lsavoie , what do you think?

Reply:
I would say 5.3.5.1 since that's where the fix is.

Reply:
we need to fix that as well in "master" or during the merge (since master already had this fix 'by accident')